views/page/reporting/courseCompletions/choose-courses.html

```diff
@@ -8,6 +8,8 @@
 
 {% extends "../../../component/Page.njk" %}
 
+{% set backButton = "/reporting/course-completions/choose-organisation" %}
```

Comment:
backButton value is set, but I can't see if it is used somewhere in this PR.

Reply:
backButton is a nunjucks block and can be found in the imported `page.njk` template (https://github.com/Civil-Service-Human-Resources/lpg-management/blob/ecf261eb74dcc8e731bb54020a508837a2f698b5/views/component/Page.njk#L78)